core/swift3Action/Dockerfile

```diff
@@ -4,16 +4,24 @@ FROM buildpack-deps:trusty
 
 ENV DEBIAN_FRONTEND noninteractive
 
-# Upgrade and install Python dependencies
+# Upgrade and install Python and helpers
 RUN apt-get -y purge && \
     apt-get -y update && \
-    apt-get -y install --fix-missing python2.7 python-distribute python-pip
+    apt-get -y install --fix-missing python2.7 && \
+    apt-get -y install --fix-missing python2.7-dev && \
+    apt-get -y install --fix-missing python-distribute && \
+    apt-get -y install --fix-missing python-pip && \
+    apt-get -y install --fix-missing python-gevent && \
+    apt-get clean && \
+    pip install -U flask
 
```

Comment:
@rabbah No. I believe this was the result of incorrectly resolving a conflict (on my part) when rebasing on cc8c48f7a62ea7eefd04dbee415741987cc35510. I'll clean this up.